COVIDWatch iOS/ViewControllers/Notifications.swift

```diff
@@ -15,6 +15,13 @@ class Notifications: BaseViewController {
     var mainText = MainText(text: "Enable notifications to receive anonymized alerts when you have come into contact with a confirmed case of COVID-19.")
     var button = Button(text: "Allow Notifications")
 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        NotificationCenter.default.addObserver(
+            self, selector: #selector(nextScreenIfNotificationsEnabled),
+            name: UIApplication.willEnterForegroundNotification, object: nil)
```

Comment:
I always removed these observations in the `deinit`, but [I guess we don't need to do that since we don't support iOS 8](https://useyourloaf.com/blog/unregistering-nsnotificationcenter-observers-in-ios-9/).  Neat!